custom_modules/MongoDbController/MongoControllerHelpers.js

```diff
@@ -169,6 +169,80 @@ class MongoControllerHelpers
 
 
 
+    /* 
+     * PATCHES
+     */
+
+    static async updateOne({
+        connection,
+        findParams,
+        obj,
+        collectionName,
+        Model,
+    })
+    {
+        return new Promise(function (resolve, reject)
+        {
+            connection.getCollection({ collectionName })
+            .then(async function (collection)
+            {
+                findParams = MongoControllerHelpers.convertIdToObjectId(findParams);
+
+                // Make query
+                const validationModel = MongoControllerHelpers.getAsModel(obj, Model);
+
+                // Validation is successful or there is no validation
+                if (!validationModel.isValid || validationModel.isValid())
+                {
+                    // Update (replace the given values for the obj)
+                    const result = await collection.findOneAndUpdate(findParams, {
```

Comment:
Using `findOneAndUpdate` because I'd like to return the old (pre-update) model and new (post-update) model once the request is complete.